cmd/buildah/config.go

```diff
@@ -3,146 +3,104 @@ package main
-		},
-		cli.StringFlag{
-			Name:  "entrypoint",
-			Usage: "set `entry point` for containers based on image",
-		},
-		cli.StringSliceFlag{
-			Name:  "env, e",
-			Usage: "add `environment variable` to be set when running containers based on image (default [])",
-		},
-		cli.StringFlag{
-			Name:  "healthcheck",
-			Usage: "set a `healthcheck` command for the target image",
-		},
-		cli.StringFlag{
-			Name:  "healthcheck-interval",
-			Usage: "set the `interval` between runs of the `healthcheck` command for the target image",
-		},
-		cli.IntFlag{
-			Name:  "healthcheck-retries",
-			Usage: "set the `number` of times the `healthcheck` command has to fail",
-		},
-		cli.StringFlag{
-			Name:  "healthcheck-start-period",
-			Usage: "set the amount of `time` to wait after starting a container before running a `healthcheck` command",
-		},
-		cli.StringFlag{
-			Name:  "healthcheck-timeout",
-			Usage: "set the maximum amount of `time` to wait for a `healthcheck` command for the target image",
-		},
-		cli.StringFlag{
-			Name:  "history-comment",
-			Usage: "set a `comment` for the history of the target image",
-		},
-		cli.StringFlag{
-			Name:  "hostname",
-			Usage: "set a host`name` for containers based on image",
-		},
-		cli.StringSliceFlag{
-			Name:  "label, l",
-			Usage: "add image configuration `label` e.g. label=value",
-		},
-		cli.StringSliceFlag{
-			Name:  "onbuild",
-			Usage: "add onbuild command to be run on images based on this image. Only supported on 'docker' formatted images",
-		},
-		cli.StringFlag{
-			Name:  "os",
-			Usage: "set `operating system` of the target image",
-		},
-		cli.StringSliceFlag{
-			Name:  "port, p",
-			Usage: "add `port` to expose when running containers based on image (default [])",
-		},
-		cli.StringFlag{
-			Name:  "shell",
-			Usage: "add `shell` to run in containers",
-		},
-		cli.StringFlag{
-			Name:  "stop-signal",
-			Usage: "set `stop signal` for containers based on image",
-		},
-		cli.StringFlag{
-			Name:  "user, u",
-			Usage: "set default `user` to run inside containers based on image",
-		},
-		cli.StringSliceFlag{
-			Name:  "volume, v",
-			Usage: "add default `volume` path to be created for containers based on image (default [])",
-		},
-		cli.StringFlag{
-			Name:  "workingdir",
-			Usage: "set working `directory` for containers based on image",
+type configResults struct {
+	addHistory             bool
+	annotation             []string
+	arch                   string
+	author                 string
+	cmd                    string
+	comment                string
+	createdBy              string
+	domainName             string
+	entrypoint             string
+	env                    []string
+	healthcheck            string
+	healthcheckInterval    string
+	healthcheckRetries     int
+	healthcheckStartPeriod string
+	healthcheckTimeout     string
+	historyComment         string
+	hostname               string
+	labels                 []string
```

Comment:
missed this earlier, think this should be singular label.

Reply:
But looking further, looks like 'labels' is an internal name and the option itself is still label, so put this in the nit can be touched up later category.

Reply:
fixed